services/src/test/java/org/exoplatform/news/rest/NewsRestResourcesV1Test.java

```diff
@@ -1954,4 +1955,62 @@ public void shouldGetStagedNewsWhenCurrentUserIsAuthor() throws Exception {
     assertEquals("john", newsList.get(0).getAuthor());
   }
 
+  @Test
+  public void getNewsIllustrationTest() {
```

Comment:
UT are not related to the added code. I understand that you want to keep the same test coverage. That is fine